app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,24 @@
+class RestaurantsController < ApplicationController
+  skip_before_action :authenticate_user!
+  def index
+    if params[:query].present?
+      @restaurants = Restaurant.search_by_restaurant(params[:query]).geocoded.reverse
```

Comment:
We may want to order the list of restaurants alphabetically (as opposed to reverse order of creation). But that we can address in the next version of the code